views/home.handlebars

```diff
@@ -1,6 +1,100 @@
+<a id="menubutton">
+  <span class="icon">
+    <svg width="22px" height="13px" viewBox="0 0 22 13" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" style="display: block;">
+      <g stroke="none" stroke-width="1" fill="none" fill-rule="evenodd">
+        <g fill="#fff">
+          <path d="M0,5 L18,5 L18,7 L0,7 L0,5 L0,5 Z"></path>
+          <path d="M0,10 L18,10 L18,12 L0,12 L0,10 L0,10 Z"></path>
+          <path d="M0,0 L18,0 L18,2 L0,2 L0,0 L0,0 Z"></path>
+        </g>
+      </g>
+    </svg>
+  </span>
+</a>
+
+<div id="menucontainer">
+  <span id="closemenubutton" class="closebutton hairline"></span>
+
+  <div class="menucontent">
+
+    <ul class="menu">
+      <li><a class="menuitem" href="javascript:void(0);" data-target="medicare">Medicare</a></li>
+      <li><a class="menuitem" href="javascript:void(0);" data-target="college">College</a></li>
+      <li><a class="menuitem" href="javascript:void(0);" data-target="rebuildamerica">Rebuild America</a></li>
+      <li><a class="menuitem" href="javascript:void(0);" data-target="ssea">Social Security</a></li>
+      <li><a class="menuitem" href="javascript:void(0);" data-target="employment">Employment</a></li>
+      <li><a class="menuitem" href="javascript:void(0);" data-target="leave">12 weeks leave</a></li>
+      <li><a class="menuitem" href="javascript:void(0);" data-target="pensions">Pensions</a></li>
+      <li><a class="menuitem" href="javascript:void(0);" data-target="estate">Estate Tax</a></li>
+      <li><a class="menuitem" href="javascript:void(0);" data-target="polluter">Polluter Welfare</a></li>
+    </ul>
+
+    <a class="btn red-btn cursive" href="https://go.berniesanders.com/page/signup/volunteer-for-bernie">JOIN THE MOVEMENT</a>
+
+  </div>
+</div>
+
+<script type="text/javascript">
```

Comment:
The reason I did the menu this way is because if we used jQuery we can
animate the scroll. If you don't care about this, maybe you can make the
change you're talking about on your end.

It seemed jQuery loaded at the end of the page and I don't know handlebars
and didn't want to spend the time figuring out how to load it. If you want
to do that you can do a little $( body ).animate( { scrollTop: y } ); or
whatever ti is.

On Tue, Apr 12, 2016 at 3:45 PM Joe Kent <notifications@github.com> wrote:

> In views/home.handlebars
> <https://github.com/deadlybutter/bernie_economics/pull/7#discussion_r59440736>
> :
>
> > +      <li><a class="menuitem" href="javascript:void(0);" data-target="college">College</a></li>
> > +      <li><a class="menuitem" href="javascript:void(0);" data-target="rebuildamerica">Rebuild America</a></li>
> > +      <li><a class="menuitem" href="javascript:void(0);" data-target="ssea">Social Security</a></li>
> > +      <li><a class="menuitem" href="javascript:void(0);" data-target="employment">Employment</a></li>
> > +      <li><a class="menuitem" href="javascript:void(0);" data-target="leave">12 weeks leave</a></li>
> > +      <li><a class="menuitem" href="javascript:void(0);" data-target="pensions">Pensions</a></li>
> > +      <li><a class="menuitem" href="javascript:void(0);" data-target="estate">Estate Tax</a></li>
> > +      <li><a class="menuitem" href="javascript:void(0);" data-target="polluter">Polluter Welfare</a></li>
> > +    </ul>
> > +
> > +    <a class="btn red-btn cursive" href="https://go.berniesanders.com/page/signup/volunteer-for-bernie">JOIN THE MOVEMENT</a>
> > +
> > +  </div>
> > +</div>
> > +
> > +<script type="text/javascript">
>
> I'm not really a fan of the approach going on here. First I'd prefer we
> stick with using jquery, but more importantly if you assign an ID to an
> HTML element you can have an href link to it. There isn't any needed for
> this JS logic that I can see.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/deadlybutter/bernie_economics/pull/7/files/f1df3aeb6f96d71b51017abf63c4620a37ac1ddb#r59440736>
>